templates/patterns/alert/alert.ui_patterns.yml

```diff
@@ -44,4 +44,4 @@ alert:
       type: text
       label: 'Icon name'
       description: 'Add an icon to the alert. Optional.'
-      preview: 'alarm'
+      preview: 'lock'
```

Comment:
Why this change?

<img width="146" alt="image" src="https://user-images.githubusercontent.com/8488617/158226686-d3efc99f-5705-490a-a153-63ca9b35bf51.png">

Can we pick a more meaningful icon.